cvat/apps/engine/migrations/0026_merge_20200417_1635.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 2.2.10 on 2020-03-22 19:38
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+
+    dependencies = [
+        ('engine', '0025_auto_20200324_1222'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='labeledshape',
+            name='annotation_type',
+            field=models.CharField(choices=[('Auto', 'AUTO'), ('Manual', 'MANUAL')], default='Manual', max_length=16, null=True),
+        ),
+        migrations.AlterField(
+            model_name='labeledshape',
+            name='type',
+            field=models.CharField(choices=[('rectangle', 'RECTANGLE'), ('polygon', 'POLYGON'), ('polyline', 'POLYLINE'), ('points', 'POINTS'), ('cuboid', 'CUBOID')], max_length=16),
```

Comment:
Why do we have the migration for "type" field here?

Reply:
Had a migration conflict. So I merged migrations to resolve it.

Reply:
@Priya4607 , we should have the old migration file and plus new one with only your changes.

Reply:
When I create a new migration, I get the `labeledshape` model changes added. I'm not sure how to include my changes alone. Could you provide some suggestions on how to proceed?

Reply:
@Priya4607 , it was a problem with develp branch. It missed some migrations. I have fixed that. Could you please update and create a new migration?